### PR TITLE
Applied comments from @csrwng to create

### PR DESCRIPTION
**Pull request by soltysh**
@mfojtik ptal

### CODE REVIEW

File: pkg/create/templates/docker.go [hunk 1]
```diff
@@ -11,8 +11,8 @@ FROM openshift/base-centos7
 # ENV BUILDER_VERSION 1.0
 
 # TODO: Set labels used in OpenShift to describe the builder image
-#LABEL k8s.io/description="Platform for building xyz" \
-#      k8s.io/display-name="builder x.y.z" \
+#LABEL io.k8s.description="Platform for building xyz" \
+#      io.k8s.display-name="builder x.y.z" \
```
Comment (bparees) [inline pkg/create/templates/docker.go]: hmm... I thought k8s was still using kis.io/display-name, was this confirmed somewhere?

Comment (soltysh) [inline pkg/create/templates/docker.go]: I'm using what @mfojtik is using in his latest changes, eg. https://github.com/openshift/sti-nodejs/pull/51

Comment (mfojtik) [inline pkg/create/templates/docker.go]: for annotations we use 'k8s.io/*' and for LABELS (which are not annotations) we follow Docker 'io.k8s.*'

**Comment by bparees** [comment]
lgtm.  [merge]

**Pull request merged**